build-scripts/rebuild_libs.rs

```diff
@@ -0,0 +1,82 @@
+/*
+* This is a script to clobber the {desktop, ios, android} directories
+* and rebuild libs via the build-all.sh script.
+*/
+use std::fs;
+use std::path::Path;
+use std::env;
+use std::process::Command;
+use std::io::prelude::*;
+
+#[allow(unused_must_use)]
+fn check_version() -> bool {
+
+    let mut clobber_needed: bool = false;
+    let root = Path::new("../libs");
+
+    assert!(env::set_current_dir(&root).is_ok());
+
+    let mut file = fs::File::open(".local_libs_version").expect("Counld not rebuild libs. Delete your folders for desktop, ios, and android, then manually run libs/build-all.sh \n");
```

Comment:
All string constants here should be named constants at the file level.

This function should take the names/paths as parameters.

Reply:
"All string constants here should be named constants at the file level." I don't know what this means